code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -229,3 +218,45 @@
 	throwforce = 0
 	breakouttime = 0
 	cuffsound = 'sound/weapons/cablecuff.ogg'
+
+//////////////////////////////
+// MARK: CRAFTING
+//////////////////////////////
+/obj/item/restraints/handcuffs/cable/attackby(obj/item/I, mob/user as mob, params)
+	..()
+	// Don't allow borgs to send their their ziptie module to the shadow realm.
+	if(istype(src, /obj/item/restraints/handcuffs/cable/zipties/cyborg))
+		return
+
+	if(istype(I, /obj/item/stack/rods))
+		var/obj/item/stack/rods/R = I
+		if(!R.use(1))
+			to_chat(user, "<span class='warning'>[R.amount > 1 ? "These rods" : "This rod"] somehow can't be used for crafting!</span>")
+			return
+
+		var/obj/item/wirerod/W = new /obj/item/wirerod
+		if(user.unEquip(src) || !remove_item_from_storage(user))
```

Comment:
As shown in discord, do:
```js
if(!unEquip(src)
	return
```
It is to check if you can even use the cable restraints for crafting